### PR TITLE
Update dashboard cli to remove dashboard_image option [skip ci]

### PR DESCRIPTION
**Pull request by IsaacYangSLA**

### CODE REVIEW

File: docs/user_guide/dashboard_api.rst [hunk 1]
```diff
@@ -75,8 +75,6 @@ Running ``nvflare dashboard -h`` shows all available options.
     -p PORT, --port PORT  port to listen
     -f FOLDER, --folder FOLDER
                             folder containing necessary info (default: current working directory)
-    -i DASHBOARD_IMAGE, --dashboard_image DASHBOARD_IMAGE
-                            container image for running dashboard
     --passphrase PASSPHRASE
```
Comment (YuanTingHsieh) [inline docs/user_guide/dashboard_api.rst:78]: this only updates the documentation, have you made the changes in code?

**Review by nvkevlu — APPROVED** [review]
Thanks, LGTM

File: docs/user_guide/dashboard_api.rst (hunk 1 above)
Comment (IsaacYangSLA) [reply docs/user_guide/dashboard_api.rst:78]: Codes were changed in #916 .

**Review by YuanTingHsieh — APPROVED** [review]

**Pull request merged**